pkgs/development/python-modules/mkdocs-jupyter/default.nix

```
@@ -0,0 +1,55 @@
+{ buildPythonPackage
+, fetchpatch
+, fetchPypi
+, ipykernel
+, jupytext
+, lib
+, lxml
+, mkdocs
+, mkdocs-material
+, nbconvert
+, pygments
+, pytestCheckHook
+, pytest-cov
+, pyyaml
+, requests
+, stdlib-list
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-jupyter";
+  version = "0.22.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-WFzGm+pMufr2iYExl43JqbIlCR7UtghPWrZWUqXhIYU=";
```

I'm not sure about that, but I believe you should use `hash` rather than `sha256` when using SRI hashes.
```suggestion
    hash = "sha256-WFzGm+pMufr2iYExl43JqbIlCR7UtghPWrZWUqXhIYU=";
```

---

done